src/hooks/job-schedules/use-job-schedules.js

```diff
@@ -0,0 +1,26 @@
+import { useDataQuery } from '@dhis2/app-runtime'
+
+const key = 'schedules'
```

Comment:
I think it'd be easier to read the code if this variable didn't exist.

Reply:
Yeah the reason I'm using this is because runtime requires a key to nest the data under, but I want to get rid of the nesting from the returned data. So I'm using this to ensure a clear link between the query and the destructuring. If I remove this variable I feel the link wouldn't be as clear (line 5 and 14, 15). Or is that not what you mean?

Reply:
What I meant is that I think it's not really that necessary to point out the link (as that's something you need to know anyway when using the app runtime), also considering [this](https://github.com/dhis2/scheduler-app/pull/495/files/107a212f5527d8b93fb2e0fb3438101ee74523d0#r1181481449) suggestion:

```jsx
const query = {
    schedules: {
        resource: 'scheduler',
    },
}

const useJobSchedules = () => {
    const fetch = useDataQuery(query)
    const schedules = fetch.data?.schedules

    // Remove nesting from data and move the id up
    if (schedules?.map) {
        const data = schedules.map((job) => {
            const id = job.sequence?.[0]?.id
            return { ...job, id }
        })

        return { ...fetch, data }
    }

    return fetch
}
```

But as I said, this is just a suggestion, feel free to leave it the way it is if you feel otherwise. It's just a tiny code snippet anyway

Reply:
> What I meant is that I think it's not really that necessary to point out the link (as that's something you need to know anyway when using the app runtime)

True, you kind of have know about that when using the app-runtime. But I feel that making it extra clear isn't bad. Makes it very obvious when refactoring. It's a bit ugly, but I prefer ugly and clear so long as it helps others understand the code.

Also, just using the `fetch.data` in the if is on purpose. Since I modify the returned data I want to make sure that that condition applies whenever data is truthy. Just so that the hook never returns unmodified data accidentally.

Reply:
I'll resolve this, but let me know if you have any remarks on the approach.